.github/workflows/ci.yml

```diff
@@ -89,3 +89,37 @@ jobs:
           # This restores requirements.txt so that @actions/cache
           # can use it for determining the cache key.
           git checkout ${GITHUB_SHA}
+  gnomad_qc:
+    name: gnomad_qc checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gnomad_methods
+        uses: actions/checkout@v2
+        with:
+          path: gnomad_methods
+      - name: Checkout gnomad_qc
+        uses: actions/checkout@v2
+        with:
+          repository: broadinstitute/gnomad_qc
```

Comment:
For my knowledge, why does this one need the repository, but gnomad_methods doesn't. Is it because this is within gnomad_methods?

Reply:
Correct. If no repository is specified, the checkout action uses the one in the workflow's context.

* https://github.com/actions/checkout#usage
* https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Reply:
Great, thank you!